cranelift/simplejit/src/backend.rs

```diff
@@ -729,16 +755,9 @@ impl RelocSink for SimpleJITRelocSink {
     }
 }
 
+#[derive(Default)]
 struct SimpleJITStackMapSink {
-    pub stack_maps: Vec<StackMapRecord>,
-}
-
-impl SimpleJITStackMapSink {
-    pub fn new() -> Self {
-        Self {
-            stack_maps: Vec::new(),
-        }
-    }
+    stack_maps: Vec<StackMapRecord>,
```

Comment:
`stack_maps` should be `pub` as well

Reply:
Same here except it doesn't seems to be used at all.